### DISCUSSION TITLE
ERR64

### DISCUSSION THREAD

**Discussion by AnrivaDeloitte** [OP-TED/eForms-SDK#1085] [Q&A] [answered]
Hi,

can you please clarify what does ERR64 mean?

Thanks a lot.

**Comment by YvesJo** [comment]
Hi,
Could you please provide further information about the encountered issue?
Thanks

**Reply by YvesJo** [reply, marked as answer]
Your request has been processed by the Helpdesk who couldn't reproduce this non-SDK related issue.

**Comment by AnrivaDeloitte** [comment]
Hi,

to clarify what type of error some users find, we copy XML files.
[ERR64.zip](https://github.com/user-attachments/files/18423477/ERR64.zip)

**Reply by YvesJo** [reply]
For this non SDK related question please get in touch with the helpdesk
